src/Appwrite/Usage/Calculators/Database.php

```diff
@@ -59,13 +61,14 @@ protected function createPerPeriodMetric(string $projectId, string $metric, int
 
         // Required for billing
         if ($monthly) {
-            $time = strtotime("first day of the month");
+            $time = DateTime::createFromFormat('Y-m-d\TH:i:s.v', \date('Y-m-01\T00:00:00.000'))->format(DateTime::RFC3339);
```

Comment:
Would have been great if we had a single consistent class to handle dates. We could also address later with @fogelito